### ISSUE TITLE
Infer state from Zip code

### ISSUE THREAD

**Issue by Frijol** [edgi-govdata-archiving/ECHO-Cross-Program#134]
Notebook: https://colab.research.google.com/github/edgi-govdata-archiving/ECHO-Cross-Program/blob/main/ECHO-Cross-Programs.ipynb#scrollTo=LH-8VrSTaRLN

If you are running the Notebook on a Zip code, specifying State (cell 3) is not required. However, the "state" field is used in creating graph titles. This means that it's easy to accidentally create misleading graphics, showing mislabeled/invalid locations. For example:

![image](https://user-images.githubusercontent.com/454690/136866338-3f78d644-75f8-4b68-82b1-110de7d9490c.png)

(The Zip code specified is in Washington, but Alabama is selected by default in the State dropdown, so the graph title shows an invalid location.)

There are two possible solutions:
1. Infer the "state" field from the Zip code so that it is correct in the graphs
2. Don't display the state when analyzing by Zip code

Option 1 seems better to me even though it adds a bit more code complexity, since it produces a more usable end result. Any opinions on that preference @ericnost @shansen5 ?

**Comment by shansen5** [comment]
Yes @byvya1505, thank you for offering to help on this.  Note that this notebook imports our ECHO_modules code, and you may need to look into that.
https://github.com/edgi-govdata-archiving/ECHO_modules
I haven't yet looked to see if there is a resource for mapping from zip codes to states.
